english/monkey_hat/monkey_hats.md

```diff
@@ -0,0 +1,4 @@
+You see a monkey with a red hat staring at you...You are taken aback, but curious. The monkey appears to grin and beckon you. You
+
+[Follow](./follow/follow.md)
+[Go back to bed in fear!](./done/sleep.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.